pysgrid/sgrid.py

```diff
@@ -252,6 +270,451 @@ def _save_common_components(self, nc_file):
                     dataset_grid_var.axes = ' '.join(axes)
         return grid_vars
 
+    def _get_grid_vars(self, name):
+        if name is 'node':
+            return (self.node_lon, self.node_lat)
+        elif name is 'center':
+            return (self.center_lon, self.center_lat)
+        elif name is 'edge1':
+            return (self.edge1_lon, self.edge1_lat)
+        elif name is 'edge2':
+            return (self.edge2_lon, self.edge2_lat)
+        else:
+            raise ValueError('Invalid grid name')
+
+    def _get_efficient_slice(self, name, indices=None, lon_bounds=None, lat_bounds=None):
```

Comment:
and (I think) the idea here is that this is a subset, yes, but it's a subset that is not specified by the caller (at least not in index space, but rather computed to be the minimal set required for the points being interpolated to.